Sources/ProjectDescription/SettingsTransformers.swift

```diff
@@ -0,0 +1,147 @@
+import Foundation
+
+extension Dictionary {
+    mutating func merge(with dict: [Key: Value]) {
+        dict.forEach { self[$0.key] = $0.value }
+    }
+
+    func merging(with dict: [Key: Value]) -> Dictionary {
+        var copy = self
+        copy.merge(with: dict)
+        return copy
+    }
```

Comment:
Those methods may not be needed - `Dictionary` already has them built into the standard language :)

- https://developer.apple.com/documentation/swift/dictionary/3127169-merge
- https://developer.apple.com/documentation/swift/dictionary/3127173-merging

Reply:
@kwridan although the stantard library has `merge` and `merging`, they still need the `combine` closure in case there's a duplicate key. I am going to change the internal implementation of the extension to use the standard library and automatically pick the second values in case there's a duplicate.